shared/react-native/ios/fastlane/Fastfile

```diff
@@ -27,12 +27,17 @@ platform :ios do
   lane :beta do
   version = get_version_number
     increment_build_number({
-      build_number: latest_testflight_build_number(version: version) + 2
+      build_number: latest_testflight_build_number(version: version) + 3
     })
 
     # match(type: "appstore") # more information: https://codesigning.guide
     gym(scheme: "Keybase") # Build your app - more options available
-    pilot
+    pilot(
+        changelog: changelog_from_git_commits(
+        pretty: '%h %s',
+        between: ["HEAD^^^^^", "HEAD"]
```

Comment:
Do we need to set `include_merges`? (based on [here](https://docs.fastlane.tools/actions/changelog_from_git_commits/))

Reply:
i don't think so. primarily we want the commit hash and thats it, i don't think we need to filter differently than the default